src/components/accessories/patientNewBill/NewBillSide.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import SmallButton from "../smallButton/SmallButton";
+import TextButton from "../textButton/TextButton";
+import "./styles.scss";
+
+interface IOwnProps {
+  handlePaymentDialog: (value: boolean) => void;
+}
+
+const NewBillSide: FC<IOwnProps> = ({ handlePaymentDialog }) => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <div className="two-row">
+        <span>Total:</span>
+        <span>120$</span>
+      </div>
+      <div className="two-row">
+        <span>Paid:</span>
+        <span>0$</span>
+      </div>
+      <div className="xs-divider"></div>
+      <div className="two-row">
+        <span>To be paid:</span>
+        <span>120$</span>
+      </div>
+      <div>
+        <SmallButton
+          onClick={() => {
+            handlePaymentDialog(true);
+          }}
+        >
+          {t("bill.pay")}
+        </SmallButton>
+      </div>
+      <div className="lg-divider"></div>
```

Comment:
Little UX trick: once the payment is complete, make the payment button disabled with a green check icon near the text "Payed".